package.json

```
@@ -7,7 +7,7 @@
     "screeps": "bin/screeps.js"
   },
   "scripts": {
-    "postinstall": "cd `node_modules` && npm explore @screeps/driver -- npx webpack"
+    "postinstall": "cd \"node_modules\" && npm explore @screeps/driver -- npx webpack"

```

```suggestion
    "postinstall": "cd node_modules && npm explore @screeps/driver -- npx webpack"
```
Pretty sure these quotes are not necessary.

---

I just went with their style, but both are fine.